lib/ui/view/organization/organization_settings/organization_icon.dart

```diff
@@ -0,0 +1,106 @@
+import 'package:flutter/material.dart';
+import 'package:stacked/stacked.dart';
+import 'package:hng/constants/app_strings.dart';
+import 'package:hng/models/organization_model.dart';
+import 'package:hng/ui/shared/colors.dart';
+import 'package:hng/ui/shared/long_button.dart';
+import 'package:hng/ui/shared/styles.dart';
+import 'package:hng/ui/shared/zuri_appbar.dart';
+import 'package:hng/ui/shared/zuri_loader.dart';
+import 'package:hng/ui/view/organization/organization_settings/organization_settings_view_model.dart';
+
+class OrganizationLogo extends StatelessWidget {
+  final OrganizationModel org;
+  const OrganizationLogo({Key? key, required this.org}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return ViewModelBuilder<OrganizationSettingsViewModel>.reactive(
+      viewModelBuilder: () => OrganizationSettingsViewModel(),
+      builder: (context, model, child) {
+        return Scaffold(
```

Comment:
Please add a screenshot for this particular widget.
I don't get why an appBar is here